input/resources/OperationDefinition-bulk-submit-data.json

```diff
@@ -0,0 +1,75 @@
+  },
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+      "valueInteger": 0
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+      "valueCode": "trial-use"
+    }
+  ],
+  "url": "http://hl7.org/fhir/us/davinci-deqm/OperationDefinition/bulk-submit-data",
+  "version": "4.0.1",
+  "name": "Bulk Submit Data",
+  "status": "draft",
+  "kind": "operation",
+  "date": "2022-11-09T10:53:32-05:00",
+  "publisher": "The MITRE Corporation",
+  "description": "The bulk submit-data operation sends a bulk import request based on the data requirements of a FHIR Measure. This operation is modeled off the existing draft of the [Bulk Data Import Ping and Pull Approach](https://github.com/smart-on-fhir/bulk-import/blob/master/import-pnp.md).",
+  "code": "submit-data",
+  "resource": [
+    "Measure"
+  ],
+  "system": false,
+  "type": true,
+  "instance": true,
+  "parameter": [
+    {
+      "name": "measureReport",
+      "use": "in",
+      "min": 1,
+      "max": "1",
+      "documentation": "The measure report being submitted",
+      "type": "MeasureReport"
+    },
```

Comment:
To also support the immediate $submit-data (which I do not recommend), we would need to add the "resource" 0..* "in" parameter from R4 $submit-data. There is not a way to have "invariant" expressions with OperationDefinition (as is done with StructureDefinitions) to say you shall not use "resources" if you are using async, and shall not use exportUrl if doing immediate. That is another reason to keep the OperationDefinition fpr $bulk-submit-data separate.